lib/kubeclient/common.rb

```diff
@@ -66,6 +71,15 @@ def initialize_client(
       end
     end
 
+    def method_missing(method_sym, *args, &block)
```

Comment:
Always define `respond_to_missing?` when implementing `method_missing`... https://robots.thoughtbot.com/always-define-respond-to-missing-when-overriding